tortoise/backends/base/db_url.py

```diff
@@ -0,0 +1,73 @@
+import urllib.parse as urlparse
+import uuid
+
+from tortoise.backends.asyncpg.client import AsyncpgDBClient
+from tortoise.backends.sqlite.client import SqliteClient
+
+urlparse.uses_netloc.append('postgres')
+urlparse.uses_netloc.append('sqlite')
+DB_LOOKUP = {
+    'postgres': {
+        'client': AsyncpgDBClient,
+        'vars': {
+            'path': 'database',
+            'hostname': 'host',
+            'port': 'port',
+            'username': 'user',
+            'password': 'password',
+        },
+        'test_defaults': {
+            'single_connection': True
+        },
+    },
+    'sqlite': {
+        'client': SqliteClient,
+        'skipfirstchar': False,
```

Comment:
I don't see why you would want to write this param without splitting `_` - `skip_first_char` seems to be clearer